rust-connectors/sinks/sql/src/opt.rs

```diff
@@ -0,0 +1,43 @@
+use clap::Parser;
+use fluvio_connectors_common::opt::CommonConnectorOpt;
+use schemars::JsonSchema;
+use serde::Deserialize;
+use std::str::FromStr;
+use url::Url;
+
+/// The JSON SQL Connector for Fluvio.
+#[derive(Parser, Debug, JsonSchema, Clone)]
+#[clap(settings = &[clap::AppSettings::DeriveDisplayOrder])]
+pub struct SqlConnectorOpt {
+    /// The connection string for the SQL database
+    ///
+    /// Ex: postgres://user:password@localhost:5432/database_name
+    #[clap(long, env = "FLUVIO_DATABASE_URL", hide_env_values = true)]
+    pub database_url: Url,
+
+    #[clap(long, env = "FLUVIO_HUB_URL", hide_env_values = true)]
+    pub hub_url: Option<Url>,
+
+    #[clap(flatten)]
+    #[schemars(flatten)]
+    pub common: CommonConnectorOpt,
+
+    #[clap(long)]
+    pub transform: Vec<TransformOpt>,
+}
+
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Parser, Debug, JsonSchema, Clone, Deserialize)]
+pub struct TransformOpt {
```

Comment:
I think that we could move this with all the `hub download` logic to a new crate

Reply:
There are such plans but we will do it a bit later once hub-related repo and crates are settled down.